### ISSUE TITLE
can someone build it for me i get errors :/ im new to vs

### ISSUE THREAD

**Issue by RXVAN** [0xd4d/dnSpy#1345]
Before clicking `Submit new issue`, try the latest build [![](https://github.com/0xd4d/dnSpy/workflows/GitHub%20CI/badge.svg)](https://github.com/0xd4d/dnSpy/actions)

- What version did you use?
newest???? idk
- Write down all steps needed to reproduce this issue. Also attach any needed files.
just open the sln

**Comment by RXVAN** [comment]
![image](https://user-images.githubusercontent.com/58669904/70870360-4e6dfb80-1f8a-11ea-8b08-9110d494e67d.png)

that is all i get :/

**Comment by Aeternus185** [comment]
Same problem :S

**Comment by Andis59** [comment]
How did you get the project? Download zip file or Git clone?
If you used download zip then you need to right-click on the zip file and select Properties and then check Unblock. Then you can unzip it...

**Comment by RXVAN** [comment]
yeah i used the zip but i didnt see the option :/ then i tried to clone it and got different errors :/

**Comment by Andis59** [comment]
Reading my post I see that have missed to mention that you need to do this in **Explorer**!
After unblocking the zip file and unzip it or cloning the project I could open the solution in Visual Studio.

I couldn't compile it since I haven't .NET 4.8 installed...

What errors do you get?

**Comment by 0xd4d** [comment]
https://github.com/0xd4d/dnSpy/releases
